com.microsoft.mrtk.windowsspeech/Subsystems/WindowsPhraseRecognitionSubsystem.cs

```diff
@@ -68,6 +74,13 @@ public override void Start()
                     keywordRecognizer.OnPhraseRecognized += Recognizer_OnPhraseRecognized;
                     keywordRecognizer.Start();
                 }
+#if MSFT_OPENXR_1_5_0_OR_NEWER
+                if (selectKeywordRecognizer != null)
+                {
+                    selectKeywordRecognizer.OnPhraseRecognized += Recognizer_OnPhraseRecognized;
```

Comment:
Will this (and the registration for `keywordRecognizer`) result in multiple identical event handlers registered if I stop/start the subsystem multiple times? It looks like deregistration happens in `Destroy`

Reply:
Thanks for the catch! Just fixed the issue.